ambry-api/src/main/java/com.github.ambry/config/StoreConfig.java

```diff
@@ -141,6 +155,11 @@ public StoreConfig(VerifiableProperties verifiableProperties) {
     storeEnableCompaction = verifiableProperties.getBoolean("store.enable.compaction", false);
     storeCompactionCheckFrequencyInHours =
         verifiableProperties.getIntInRange("store.compaction.check.frequency.in.hours", 7 * 24, 1, Integer.MAX_VALUE);
+    storeCompactionPolicyFactory = verifiableProperties.getString("store.compaction.policy.factory",
+        "com.github.ambry.store.DefaultCompactionPolicyFactory");
+    storeMinLogSegmentCountToReclaimToTriggerCompaction =
+        verifiableProperties.getInt("store.min.log.segment.count.to.reclaim.to.trigger.compaction", 1);
```

Comment:
This would depend on the max no of log segments right. Else how do we set the max value ?